main.py

```diff
@@ -1,10 +1,37 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+    for nick_name, info in data.items():
+        race, is_created = Race.objects.get_or_create(
```

Comment:
You do not utilize the `is_created` method, you could omit it with `_`. The same relates to the `is_created` on line 22.